### ISSUE TITLE
wrap is not a function

### ISSUE THREAD

**Issue by prescottprue** [firebase/firebase-functions-test#2]
### Version info

**firebase-functions-test:**
`0.1.0`

**firebase-functions:**
`1.0.0`

**firebase-admin:**
`5.11.0`

### Test case

Following [the unit testing guide](https://firebase.google.com/docs/functions/unit-testing) I try to do the following:

```js
const test = require('firebase-functions-test')()

test.wrap(myFunctions.someFunc)
```

### Steps to reproduce

1. Write a function test that includes `test.wrap`
1. Run the test through mocha - error is thrown: `TypeError: test.wrap is not a function`

### Expected behavior

Based on the docs [and this example](https://github.com/firebase/functions-samples/blob/master/quickstarts/uppercase/functions/test/test.offline.js#L82) it was expected that `test.wrap` would be a function. I couldn't tell if it was an issue with the docs or with the exports of the library itself.

### Actual behavior

I needed to do `test.features.wrap` to access the wrap function

**Comment by btrautmann** [comment]
I'm currently having the same issue. I will continue to debug by pulling down the quickstarts project and attempting to run tests with the `firebase-functions-test` SDK.

Can someone confirm that this SDK _is_ indeed usable for testing **Cloud Firestore** functions?

**Comment by nicolasgarnier** [comment]
@laurenzlong PTAL. Either an issue with the sample or this package.

FYI on https://github.com/firebase/functions-samples/issues/358 @Westbrook said he's able to use `test.features.wrap` instead of `test.wrap`

**Comment by lampietti** [comment]
Here is the actual ```test``` struct with firebase-functions-test v0.1.0

```
{ features:
   { mockConfig: [Function: mockConfig],
     wrap: [Function: wrap],
     makeChange: [Function: makeChange],
     analytics:
      { makeAnalyticsEvent: [Function: makeAnalyticsEvent],
        exampleAnalyticsEvent: [Function: exampleAnalyticsEvent] },
     auth:
      { makeUserRecord: [Function: makeUserRecord],
        exampleUserRecord: [Function: exampleUserRecord] },
     crashlytics:
      { makeIssue: [Function: makeIssue],
        exampleIssue: [Function: exampleIssue] },
     database:
      { makeDataSnapshot: [Function: makeDataSnapshot],
        exampleDataSnapshot: [Function: exampleDataSnapshot],
        exampleDataSnapshotChange: [Function: exampleDataSnapshotChange] },
     firestore:
      { makeDocumentSnapshot: [Function: makeDocumentSnapshot],
        exampleDocumentSnapshot: [Function: exampleDocumentSnapshot],
        exampleDocumentSnapshotChange: [Function: exampleDocumentSnapshotChange],
        objectToValueProto: [Function: objectToValueProto] },
     pubsub:
      { makeMessage: [Function: makeMessage],
        exampleMessage: [Function: exampleMessage] },
     storage:
      { makeObjectMetadata: [Function: makeObjectMetadata],
        exampleObjectMetadata: [Function: exampleObjectMetadata] } },
  cleanup: [Function: cleanup] }
```

So, it is not only with the ```wrap()``` function
And yes, it is working by replacing it by ```test.features.*``` but i don't think it is the expected behavior, since all samples are failing (including documentation ones in ["Constructing Test Data"](https://firebase.google.com/docs/functions/unit-testing#constructing_test_data)

**Comment by laurenzlong** [comment]
Thanks for the quick feedback! Just released v0.1.1, which fixes this. The [sample](https://github.com/firebase/functions-samples/blob/master/quickstarts/uppercase/functions/test/test.offline.js#L82) still fails because there's a bug in the index.js code, and it's also out of sync with the test code, but we'll get it fixed soon.

**Comment by prescottprue** [comment]
@laurenzlong What are thoughts on getting some tests in place for the top level exports? I started [this PR](https://github.com/firebase/firebase-functions-test/pulls), but if you think its overkill we can just close it.
